### PR TITLE
news: add news for 0.29.1

### PR DESCRIPTION
**Pull request by ajeddeloh**

### CODE REVIEW

**Comment by arithx** [comment]
Assuming this is dependent on the merge of #676

**Comment by ajeddeloh** [comment]
Yes, thought I'd speculatively put it up so I can just do the whole release in one once it merges (waiting on a kola run).

File: NEWS [hunk 1]
```diff
@@ -1,3 +1,10 @@
+06-Dec-2018 IGNITION v0.29.1
+
+  Bug Fixes:
+
+    - Do not follow symlinks if they are the last path element when writing
```
Comment (bgilbert) [inline NEWS]: "if they are the last path element when writing" seems a bit awkward.  "When writing files, directories, or links, do not follow symlinks if they are the last path element"?

**Comment by ajeddeloh** [comment]
Fixed

**Pull request merged**